src/api/app/controllers/webui/project_controller.rb

```diff
@@ -24,6 +24,8 @@ class Webui::ProjectController < Webui::WebuiController
   before_filter :load_releasetargets, :only => [ :show, :incident_request_dialog, :release_repository_dialog ]
   prepend_before_filter :lockout_spiders, :only => [:requests, :rebuild_time, :buildresults, :maintenance_incidents]
 
+  after_action :verify_authorized, :only => :save_new
```

Comment:
If I understand correctly the after_action callback runs after processing the action. So I wonder if we shouldn't use before_action instead.